algorithmic_efficiency/init_utils.py

```diff
@@ -0,0 +1,11 @@
+import math
+
+from torch import nn
+
+
```

Comment:
Maybe it makes sense to add a comment with the [reference](https://jax.readthedocs.io/en/latest/_autosummary/jax.nn.initializers.lecun_normal.html) to the corresponding Jax initialization function here?

Reply:
That's a great point. I will add that.